pipeline/src/helpers/prismic.ts

```diff
@@ -45,7 +66,7 @@ export const getPrismicDocuments = async <T>({
   contentTypes,
   after,
 }: GetPrismicDocumentsParams): Promise<PrismicPage<T>> => {
-  const docs = await client.getByType(contentTypes[0], {
+  const docs = await client.get({
```

Comment:
@jamieparkinson fyi; just realised `webcomics` were still being skipped on. Now using simple `get` and using the predicates to control which content types to fetch.